lib/src/lib.rs

```diff
@@ -21,11 +21,12 @@ pub mod builder;
 pub mod consts;
 pub mod input;
 pub mod mem_db;
+pub mod optimism;
 
 pub use zeth_primitives::transactions::{ethereum::EthereumTxEssence, optimism::OptimismTxEssence};
 
 /// call forget only if running inside the guest
 pub fn guest_mem_forget<T>(_t: T) {
-    #[cfg(target_os = "zkvm")]
-    core::mem::forget(_t)
+    // #[cfg(target_os = "zkvm")]
+    // core::mem::forget(_t)
```

Comment:
It does safe quite a few cycles and I know @hashcashier was a big fan of this 😄 
If there is a particular reason for removing this we should remove the entire function instead or uncomment it.

Reply:
Not sure how that slipped in there. I will re-enable. (Done.)